build/localmode/RunSamples.cmd

```diff
@@ -8,11 +8,6 @@ set CMDHOME=%CMDHOME:~0,-1%
 set VERBOSE=
 set USER_EXE=
 
-set CMDHOME=%~dp0
-
-@REM Remove trailing backslash \
-set CMDHOME=%CMDHOME:~0,-1%
```

Comment:
The code is already in this file at line 3, so this is removing the duplicate code.